packages/foreman/rubygem-sqlite3/rubygem-sqlite3.spec

```diff
@@ -22,6 +22,7 @@ BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 1.8.7
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.5
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+BuildRequires: gcc
```

Comment:
If we regenerate the spec I think this should be automatically done. There is an update to 1.6.9 available, so perhaps just do that? Also drops the SCL stuff.

Reply:
sqlite3 1.5.0+ introduces a dep on mini_portile2, which I didn't want to open up as a can-o-worms right now. only updating to 1.4.4 then,